From bdf1581e3de5528f397f19bfd4ca9caaf9e7fe4a Mon Sep 17 00:00:00 2001
From: Behdad Esfahbod <behdad@behdad.org>
Date: Fri, 8 Mar 2013 05:53:27 -0500
Subject: [PATCH] Fix crash with FcConfigSetCurrent(NULL)

---
 src/fccfg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fccfg.c b/src/fccfg.c
index b762376..7da50b5 100644
--- a/src/fccfg.c
+++ b/src/fccfg.c
@@ -429,7 +429,7 @@ retry:
     if (config == cfg)
 	return FcTrue;
 
-    if (!config->fonts[FcSetSystem])
+    if (config && !config->fonts[FcSetSystem])
 	if (!FcConfigBuildFonts (config))
 	    return FcFalse;
 
-- 
1.8.1.3

