diff -up fontconfig-2.15.0/conf.d/45-latin.conf.2~ fontconfig-2.15.0/conf.d/45-latin.conf
--- fontconfig-2.15.0/conf.d/45-latin.conf.2~	2023-08-09 13:32:05.000000000 +0200
+++ fontconfig-2.15.0/conf.d/45-latin.conf	2025-01-13 03:06:49.001455807 +0100
@@ -59,6 +59,10 @@
 		<default><family>serif</family></default>
 	</alias>
 	<alias>
+		<family>Noto Serif</family>
+		<default><family>serif</family></default>
+	</alias>
+	<alias>
 		<family>Palatino Linotype</family>
 		<default><family>serif</family></default>
 	</alias>
@@ -71,6 +75,10 @@
 		<default><family>serif</family></default>
 	</alias>
 	<alias>
+		<family>Antykwa Poltawskiego</family>
+		<default><family>serif</family></default>
+	</alias>
+	<alias>
 		<family>Times New Roman</family>
 		<default><family>serif</family></default>
 	</alias>
@@ -154,6 +162,10 @@
 		<default><family>sans-serif</family></default>
 	</alias>
 	<alias>
+		<family>Noto Sans</family>
+		<default><family>sans-serif</family></default>
+	</alias>
+	<alias>
 		<family>Luxi Sans</family>
 		<default><family>sans-serif</family></default>
 	</alias>
@@ -241,6 +253,14 @@
 		<default><family>monospace</family></default>
 	</alias>
 	<alias>
+		<family>Noto Mono</family>
+		<default><family>monospace</family></default>
+	</alias>
+	<alias>
+		<family>Noto Sans Mono CJK SC</family>
+		<default><family>monospace</family></default>
+	</alias>
+	<alias>
 		<family>Terminal</family>
 		<default><family>monospace</family></default>
 	</alias>
diff -up fontconfig-2.15.0/conf.d/60-latin.conf.2~ fontconfig-2.15.0/conf.d/60-latin.conf
--- fontconfig-2.15.0/conf.d/60-latin.conf.2~	2021-12-03 11:48:08.000000000 +0100
+++ fontconfig-2.15.0/conf.d/60-latin.conf	2025-01-13 03:06:49.001522733 +0100
@@ -36,6 +36,7 @@
 		<family>monospace</family>
 		<prefer>
 			<family>Noto Sans Mono</family>
+			<family>Noto Mono</family>
 			<family>DejaVu Sans Mono</family>
 			<family>Inconsolata</family>
 			<family>Andale Mono</family>
diff -up fontconfig-2.15.0/conf.d/65-nonlatin.conf.2~ fontconfig-2.15.0/conf.d/65-nonlatin.conf
--- fontconfig-2.15.0/conf.d/65-nonlatin.conf.2~	2023-03-22 12:36:46.000000000 +0100
+++ fontconfig-2.15.0/conf.d/65-nonlatin.conf	2025-01-13 03:09:41.810895921 +0100
@@ -24,13 +24,25 @@
 			<family>Sampige</family> <!-- kannada -->
 			<family>padmaa</family> <!-- gujarati -->
 			<family>Hapax Berbère</family> <!-- tifinagh -->
+			<family>Noto Sans CJK SC</family>
+			<family>Noto Sans CJK TC</family>
 			<family>MS Mincho</family> <!-- han (ja) -->
+			<family>Microsoft YaHei UI</family> <!-- han (zh-cn,zh-tw) -->
+			<family>Microsoft JhengHei UI</family> <!-- han (zh-tw) -->
+			<family>Source Han Sans</family> <!-- han (ja, ko, zh) -->
+			<family>Open Hei</family> <!--han (zh-tw) -->
+			<family>WenQuanYi Micro Hei</family> <!-- han (zh-cn,zh-tw) -->
 			<family>SimSun</family> <!-- han (zh-cn,zh-tw) -->
 			<family>PMingLiu</family> <!-- han (zh-tw) -->
+			<family>Source Han Sans</family> <!-- han (ja, ko, zh) -->
+			<family>Open Hei</family> <!--han (zh-tw) -->
+			<family>WenQuanYi Micro Hei</family> <!-- han (zh-cn,zh-tw) -->
 			<family>WenQuanYi Zen Hei</family> <!-- han (zh-cn,zh-tw) -->
 			<family>WenQuanYi Bitmap Song</family> <!-- han (zh-cn,zh-tw) -->
 			<family>AR PL ShanHeiSun Uni</family> <!-- han (ja,zh-cn,zh-tw) -->
 			<family>AR PL New Sung</family> <!-- han (zh-cn,zh-tw) -->
+			<family>AR PL UMing TW MBE</family> <!--han (zh-tw) -->
+			<family>AR PL UMing CN</family> <!--han (ja,zh-cn) -->
 			<family>ZYSong18030</family> <!-- han (zh-cn,zh-tw) -->
 			<family>HanyiSong</family> <!-- han (zh-cn,zh-tw) -->
 			<family>Hiragino Mincho ProN</family> <!-- han (ja) - macOS -->
@@ -163,12 +175,18 @@
 			<family>Sazanami Gothic</family>
 			<family>Kochi Gothic</family>
 			<family>AR PL KaitiM GB</family>
+			<family>Noto Sans Mono CJK SC</family>
+			<family>Noto Sans Mono CJK TC</family>
 			<family>MS Gothic</family> <!-- han (ja) -->
 			<family>UmePlus Gothic</family> <!-- han (ja) -->
 			<family>NSimSun</family> <!-- han (zh-cn,zh-tw) -->
 			<family>MingLiu</family> <!-- han (zh-tw) -->
-			<family>AR PL ShanHeiSun Uni</family> <!-- han (ja,zh-cn,zh-tw) -->
+			<family>Open Hei</family> <!--han (zh-tw) -->
+			<family>WenQuanYi Micro Hei Mono</family>  <!-- han (zh-cn,zh-tw) -->
+			<family>WenQuanYi Zen Hei Mono</family>  <!-- han (zh-cn,zh-tw) -->
 			<family>AR PL New Sung Mono</family> <!-- han (zh-cn,zh-tw) -->
+			<family>AR PL UMing TW MBE</family> <!-- han (zh-tw) -->
+			<family>AR PL UMing CN</family> <!-- han (ja,zh-cn) -->
 			<family>HanyiSong</family> <!-- han (zh-cn) -->
 			<family>AR PL SungtiL GB</family>
 			<family>AR PL Mingti2L Big5</family>
