diff -uraN fontconfig-2.17.1/conf.d/Makefile.am omv-fontconfig-2.17.1/conf.d/Makefile.am
--- fontconfig-2.17.1/conf.d/Makefile.am	2025-07-02 06:46:34.000000000 +0200
+++ omv-fontconfig-2.17.1/conf.d/Makefile.am	2025-09-06 00:58:23.010737261 +0200
@@ -30,6 +30,8 @@
 DOC_FILES = $(DOC_SOURCES:.in=)
 
 CONF_LINKS = \
+    10-antialias.conf \
+    10-hinting.conf \
 	10-hinting-$(PREFERRED_HINTING).conf	\
 	10-scale-bitmap-fonts.conf \
 	10-yes-antialias.conf	\
@@ -64,7 +66,9 @@
 template_DATA =				\
 	05-reset-dirs-sample.conf	\
 	09-autohint-if-no-hinting.conf	\
+    10-antialias.conf		\
 	10-autohint.conf		\
+    10-hinting.conf			\
 	10-hinting-full.conf		\
 	10-hinting-medium.conf		\
 	10-hinting-none.conf		\
@@ -94,6 +98,7 @@
 	49-sansserif.conf		\
 	50-user.conf			\
 	51-local.conf			\
+    53-monospace-lcd-filter.conf	\
 	60-generic.conf			\
 	60-latin.conf			\
 	65-fonts-persian.conf		\
