diff -Naur fontconfig-2.12.4/conf.d/45-latin.conf fontconfig-2.12.4.tpg/conf.d/45-latin.conf
--- fontconfig-2.12.4/conf.d/45-latin.conf	2016-12-02 03:22:19.020598016 +0000
+++ fontconfig-2.12.4.tpg/conf.d/45-latin.conf	2017-07-27 21:51:10.255489598 +0000
@@ -58,6 +58,10 @@
 		<default><family>serif</family></default>
 	</alias>
 	<alias>
+		<family>Noto Serif</family>
+		<default><family>serif</family></default>
+	</alias>
+	<alias>
 		<family>Palatino Linotype</family>
 		<default><family>serif</family></default>
 	</alias>
@@ -70,6 +74,10 @@
 		<default><family>serif</family></default>
 	</alias>
 	<alias>
+		<family>Antykwa Poltawskiego</family>
+		<default><family>serif</family></default>
+	</alias>
+	<alias>
 		<family>Times New Roman</family>
 		<default><family>serif</family></default>
 	</alias>
@@ -149,6 +157,10 @@
 		<default><family>sans-serif</family></default>
 	</alias>
 	<alias>
+		<family>Noto Sans</family>
+		<default><family>sans-serif</family></default>
+	</alias>
+	<alias>
 		<family>Luxi Sans</family>
 		<default><family>sans-serif</family></default>
 	</alias>
@@ -232,6 +244,14 @@
 		<default><family>monospace</family></default>
 	</alias>
 	<alias>
+		<family>Noto Mono</family>
+		<default><family>monospace</family></default>
+	</alias>
+	<alias>
+		<family>Noto Sans Mono CJK SC</family>
+		<default><family>monospace</family></default>
+	</alias>
+	<alias>
 		<family>Terminal</family>
 		<default><family>monospace</family></default>
 	</alias>
diff -Naur fontconfig-2.12.4/conf.d/60-latin.conf fontconfig-2.12.4.tpg/conf.d/60-latin.conf
--- fontconfig-2.12.4/conf.d/60-latin.conf	2016-12-02 03:22:19.020598016 +0000
+++ fontconfig-2.12.4.tpg/conf.d/60-latin.conf	2017-07-27 21:53:21.069117770 +0000
@@ -4,8 +4,9 @@
 	<alias>
 		<family>serif</family>
 		<prefer>
-			<family>Bitstream Vera Serif</family>
+			<family>Noto Serif</family>
 			<family>DejaVu Serif</family>
+			<family>Bitstream Vera Serif</family>
 			<family>Times New Roman</family>
 			<family>Thorndale AMT</family>
 			<family>Luxi Serif</family>
@@ -17,15 +18,16 @@
 	<alias>
 		<family>sans-serif</family>
 		<prefer>
-			<family>Bitstream Vera Sans</family>
+			<family>Noto Sans</family>
 			<family>DejaVu Sans</family>
-			<family>Verdana</family>
-			<family>Arial</family>
-			<family>Albany AMT</family>
+			<family>Bitstream Vera Sans</family>
 			<family>Luxi Sans</family>
 			<family>Nimbus Sans L</family>
 			<family>Nimbus Sans</family>
+			<family>Arial</family>
+			<family>Albany AMT</family>
 			<family>Helvetica</family>
+			<family>Verdana</family>
 			<family>Lucida Sans Unicode</family>
 			<family>BPG Glaho International</family> <!-- lat,cyr,arab,geor -->
 			<family>Tahoma</family> <!-- lat,cyr,greek,heb,arab,thai -->
@@ -34,16 +36,18 @@
 	<alias>
 		<family>monospace</family>
 		<prefer>
-			<family>Bitstream Vera Sans Mono</family>
+			<family>Noto Mono</family>
+			<family>Noto Sans Mono CJK SC</family>
 			<family>DejaVu Sans Mono</family>
+			<family>Bitstream Vera Sans Mono</family>
 			<family>Inconsolata</family>
-			<family>Andale Mono</family>
-			<family>Courier New</family>
-			<family>Cumberland AMT</family>
 			<family>Luxi Mono</family>
 			<family>Nimbus Mono L</family>
 			<family>Nimbus Mono</family>
 			<family>Nimbus Mono PS</family>
+			<family>Andale Mono</family>
+			<family>Courier New</family>
+			<family>Cumberland AMT</family>
 			<family>Courier</family>
 		</prefer>
 	</alias>
diff -Naur fontconfig-2.12.4/conf.d/65-nonlatin.conf fontconfig-2.12.4.tpg/conf.d/65-nonlatin.conf
--- fontconfig-2.12.4/conf.d/65-nonlatin.conf	2016-07-11 02:41:05.020598016 +0000
+++ fontconfig-2.12.4.tpg/conf.d/65-nonlatin.conf	2017-07-27 21:55:49.528067277 +0000
@@ -23,13 +23,20 @@
 			<family>Sampige</family> <!-- kannada -->
 			<family>padmaa</family> <!-- gujarati -->
 			<family>Hapax Berbère</family> <!-- tifinagh -->
+			<family>Noto Sans CJK SC</family>
+			<family>Noto Sans CJK TC</family>
 			<family>MS Mincho</family> <!-- han (ja) -->
 			<family>SimSun</family> <!-- han (zh-cn,zh-tw) -->
 			<family>PMingLiu</family> <!-- han (zh-tw) -->
+			<family>Source Han Sans</family> <!-- han (ja, ko, zh) -->
+			<family>Open Hei</family> <!--han (zh-tw) -->
+			<family>WenQuanYi Micro Hei</family> <!-- han (zh-cn,zh-tw) -->
 			<family>WenQuanYi Zen Hei</family> <!-- han (zh-cn,zh-tw) -->
 			<family>WenQuanYi Bitmap Song</family> <!-- han (zh-cn,zh-tw) -->
 			<family>AR PL ShanHeiSun Uni</family> <!-- han (ja,zh-cn,zh-tw) -->
 			<family>AR PL New Sung</family> <!-- han (zh-cn,zh-tw) -->
+			<family>AR PL UMing TW MBE</family> <!--han (zh-tw) -->
+			<family>AR PL UMing CN</family> <!--han (ja,zh-cn) -->
 			<family>ZYSong18030</family> <!-- han (zh-cn,zh-tw) -->
 			<family>HanyiSong</family> <!-- han (zh-cn,zh-tw) -->
 			<family>MgOpen Canonica</family>
@@ -94,15 +101,21 @@
 			<family>Sampige</family> <!-- kannada -->
 			<family>padmaa</family> <!-- gujarati -->
 			<family>Hapax Berbère</family> <!-- tifinagh -->
+			<family>Noto Sans CJK SC</family>
+			<family>Noto Sans CJK TC</family>
 			<family>MS Gothic</family> <!-- han (ja) -->
 			<family>UmePlus P Gothic</family> <!-- han (ja) -->
-			<!-- chinese fonts are actually serifed -->
-			<family>SimSun</family> <!-- han (zh-cn,zh-tw) -->
-			<family>PMingLiu</family> <!-- han (zh-tw) -->
+			<family>Microsoft YaHei UI</family> <!-- han (zh-cn,zh-tw) -->
+			<family>Microsoft JhengHei UI</family> <!-- han (zh-tw) -->
+			<family>Source Han Sans</family> <!-- han (ja, ko, zh) -->
+			<family>Open Hei</family> <!--han (zh-tw) -->
+			<family>WenQuanYi Micro Hei</family> <!-- han (zh-cn,zh-tw) -->
 			<family>WenQuanYi Zen Hei</family> <!-- han (zh-cn,zh-tw) -->
 			<family>WenQuanYi Bitmap Song</family> <!-- han (zh-cn,zh-tw) -->
 			<family>AR PL ShanHeiSun Uni</family> <!--han (ja,zh-cn,zh-tw) -->
 			<family>AR PL New Sung</family> <!-- han (zh-cn,zh-tw) -->
+			<family>AR PL UMing TW MBE</family> <!--han (zh-tw) -->
+			<family>AR PL UMing CN</family> <!--han (ja,zh-cn) -->
 			<family>MgOpen Modata</family>
 			<family>VL Gothic</family>
 			<family>IPAMonaGothic</family>
@@ -151,12 +164,18 @@
 			<family>Sazanami Gothic</family>
 			<family>Kochi Gothic</family>
 			<family>AR PL KaitiM GB</family>
+			<family>Noto Sans Mono CJK SC</family>
+			<family>Noto Sans Mono CJK TC</family>
 			<family>MS Gothic</family> <!-- han (ja) -->
 			<family>UmePlus Gothic</family> <!-- han (ja) -->
 			<family>NSimSun</family> <!-- han (zh-cn,zh-tw) -->
 			<family>MingLiu</family> <!-- han (zh-tw) -->
-			<family>AR PL ShanHeiSun Uni</family> <!-- han (ja,zh-cn,zh-tw) -->
+			<family>Open Hei</family> <!--han (zh-tw) -->
+			<family>WenQuanYi Micro Hei Mono</family>  <!-- han (zh-cn,zh-tw) -->
+			<family>WenQuanYi Zen Hei Mono</family>  <!-- han (zh-cn,zh-tw) -->
 			<family>AR PL New Sung Mono</family> <!-- han (zh-cn,zh-tw) -->
+			<family>AR PL UMing TW MBE</family> <!-- han (zh-tw) -->
+			<family>AR PL UMing CN</family> <!-- han (ja,zh-cn) -->
 			<family>HanyiSong</family> <!-- han (zh-cn) -->
 			<family>AR PL SungtiL GB</family>
 			<family>AR PL Mingti2L Big5</family>
