diff -Naur fontconfig-2.14.1/conf.d/Makefile.am fontconfig-2.14.1.tpg/conf.d/Makefile.am
--- fontconfig-2.14.1/conf.d/Makefile.am	2022-10-21 04:39:48.000000000 +0000
+++ fontconfig-2.14.1.tpg/conf.d/Makefile.am	2022-10-27 18:46:37.701913514 +0000
@@ -30,6 +30,8 @@
 DOC_FILES = $(DOC_SOURCES:.in=)
 
 CONF_LINKS = \
+	10-antialias.conf \
+	10-hinting.conf \
 	10-hinting-$(PREFERRED_HINTING).conf	\
 	10-scale-bitmap-fonts.conf \
 	10-yes-antialias.conf	\
@@ -62,7 +64,9 @@
 template_DATA =				\
 	05-reset-dirs-sample.conf	\
 	09-autohint-if-no-hinting.conf	\
+	10-antialias.conf		\
 	10-autohint.conf		\
+	10-hinting.conf			\
 	10-hinting-full.conf		\
 	10-hinting-medium.conf		\
 	10-hinting-none.conf		\
@@ -90,6 +94,7 @@
 	49-sansserif.conf		\
 	50-user.conf			\
 	51-local.conf			\
+	53-monospace-lcd-filter.conf	\
 	60-generic.conf			\
 	60-latin.conf			\
 	65-fonts-persian.conf		\
diff -Naur fontconfig-2.14.1/conf.d/Makefile.in fontconfig-2.14.1.tpg/conf.d/Makefile.in
--- fontconfig-2.14.1/conf.d/Makefile.in	2022-10-21 07:33:38.000000000 +0000
+++ fontconfig-2.14.1.tpg/conf.d/Makefile.in	2022-10-27 18:50:27.360952936 +0000
@@ -387,6 +387,8 @@
 DOC_SOURCES = README.in
 DOC_FILES = $(DOC_SOURCES:.in=)
 CONF_LINKS = \
+	10-antialias.conf \
+	10-hinting.conf \
 	10-hinting-$(PREFERRED_HINTING).conf	\
 	10-scale-bitmap-fonts.conf \
 	10-yes-antialias.conf	\
@@ -417,7 +419,9 @@
 template_DATA = \
 	05-reset-dirs-sample.conf	\
 	09-autohint-if-no-hinting.conf	\
+	10-antialias.conf		\
 	10-autohint.conf		\
+	10-hinting.conf			\
 	10-hinting-full.conf		\
 	10-hinting-medium.conf		\
 	10-hinting-none.conf		\
@@ -445,6 +449,7 @@
 	49-sansserif.conf		\
 	50-user.conf			\
 	51-local.conf			\
+	53-monospace-lcd-filter.conf	\
 	60-generic.conf			\
 	60-latin.conf			\
 	65-fonts-persian.conf		\
